lineman/app/components/navbar/navbar.scss

```diff
@@ -46,6 +46,18 @@
   a{ color: $primary-text-color; }
 }
 
+.lmo-navbar__item--user, .lmo-navbar__item--notifications {
```

Comment:
@AaronThornton00 - I think I've mentioned to you that the CSS guidelines we're really enjoying say: "Dont use nesting unless you really really have to" - I'm wondering if you need to be using nesting here?

The other thing I want to do is use mixin arguments to set the left value rather than adding more selectors.
I'll show you what I mean in another comment.